### DISCUSSION TITLE
Issue with the click-through link in alert details

### DISCUSSION THREAD

**Discussion by zhangj-923** [ccfos/nightingale#1274] [General]
在历史告警页面点击告警标题，会打开新的窗口去查看，这个是设计如此嘛？感觉可以优化一下。
![image](https://user-images.githubusercontent.com/67679447/202370576-9749c23b-3f26-469b-b6eb-0295e3602286.png)
目前在使用的版本
![image](https://user-images.githubusercontent.com/67679447/202370694-b639d1b1-b5be-4802-a404-2e4209157b83.png)

**Comment by UlricQin** [comment]
是的，故意这么设计的，相当于事件详情有个单独的url，后面告警的的时候，就可以在短信、邮件里边直接放超链接，链过来
